source/_docs/automation/trigger.markdown

```diff
@@ -435,7 +435,7 @@ automation:
 
 #### Input Datetime
 
-The Entity ID of an [Input Datetime](/integrations/input_datetime/).
+The Entity ID of an [input datetime](/integrations/input_datetime/).
```

Comment:
This refers to a name of integration and thus should be capitalized here.

Reply:
I think I saw others that were lower case, which is why I adjusted here. Will revert.